Mathlib/Analysis/Analytic/Basic.lean

```diff
@@ -337,6 +338,88 @@ theorem radius_smul_eq (p : FormalMultilinearSeries 𝕜 E F) {c : 𝕜} (hc : c
   apply eq_of_le_of_le _ radius_le_smul
   exact radius_le_smul.trans_eq (congr_arg _ <| inv_smul_smul₀ hc p)
 
+lemma norm_compContinuousLinearMap_le {p : FormalMultilinearSeries 𝕜 F G} {u : E →L[𝕜] F} {n : ℕ} :
+    ‖p.compContinuousLinearMap u n‖ ≤ ‖p n‖ * ‖u‖^n := by
+  simp only [compContinuousLinearMap]
+  apply le_trans (ContinuousMultilinearMap.norm_compContinuousLinearMap_le _ _)
+  simp
+
+theorem radius_compContinuousLinearMap_ge (p : FormalMultilinearSeries 𝕜 F G) (u : E →L[𝕜] F) :
+    p.radius / ‖u‖₊ ≤ (p.compContinuousLinearMap u).radius := by
+  by_cases h_zero : ‖u‖₊ = 0
+  · simp only [nnnorm_eq_zero] at h_zero
+    simp [h_zero]
+  simp only [radius, ENNReal.iSup_div]
+  apply iSup_mono'
+  intro r
+  use r / ‖u‖₊
+  apply iSup_mono'
+  intro C
+  use C
+  apply iSup_mono'
+  simp only [NNReal.coe_div, coe_nnnorm, exists_prop]
+  intro h1
+  constructor
+  · intro n
+    calc
+      _ ≤ (‖p n‖ * ‖u‖ ^ n) * (↑r / ‖u‖) ^ n := by
+        gcongr
+        apply norm_compContinuousLinearMap_le
+      _ ≤ _ := by
+        ring_nf
+        move_mul [← ‖u‖⁻¹ ^ n, ← ‖u‖ ^ n]
+        rw [← mul_pow, inv_mul_cancel₀]
+        · simpa using h1 n
+        · simpa using h_zero
+  · rw [ENNReal.coe_div h_zero]
+
+theorem radius_compContinuousLinearMap_le [Nontrivial F]
+    (p : FormalMultilinearSeries 𝕜 F G) (u : E ≃L[𝕜] F) :
+    (p.compContinuousLinearMap u.toContinuousLinearMap).radius ≤
+    ‖u.symm.toContinuousLinearMap‖₊ * p.radius := by
+  suffices (p.compContinuousLinearMap u.toContinuousLinearMap).radius /
+      ‖(u.symm : F →L[𝕜] E)‖₊ ≤ p.radius by
+    rw [div_eq_mul_inv] at this
+    rw [ENNReal.mul_inv_le_iff] at this
+    · convert this using 1
+      ring
+    · simpa using ContinuousLinearEquiv.norm_pos u.symm
+    · simp
+  have : p =
+      (p.compContinuousLinearMap u.toContinuousLinearMap).compContinuousLinearMap u.symm := by
+    simp [compContinuousLinearMap_comp]
+  conv => rhs; rw [this]
+  apply radius_compContinuousLinearMap_ge
```

Comment:
In this case, it's easier to just build the term you want to simplify by applying the earlier result.
```suggestion
  have := (p.compContinuousLinearMap u.toContinuousLinearMap).div_le_radius_compContinuousLinearMap
    u.symm.toContinuousLinearMap
  simp only [compContinuousLinearMap_comp, ContinuousLinearEquiv.coe_comp_coe_symm,
    compContinuousLinearMap_id] at this
  rwa [ENNReal.div_le_iff' (by simpa [DFunLike.ext_iff] using exists_ne 0) (by simp)] at this
```